website/docs/add-secure-apps/flows-stages/stages/single_logout.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Single Logout (SLO)
+---
+
+Single Logout is a security feature that logs a user out of all their applications with active sessions when they log out of authentik. It uses the OAuth2/OpenID Connect front-channel and back-channel logout specifications in combination with SAML's Single Logout specification.
+
+For example, if a user is concurrently logged into one OIDC provider and two SAML providers, when the user logs out of authentik, they will automatically be logged out of all three applications. Without SLO configured, users with active sessions across multiple providers would need to manually log out of each one individually.
+
+:::info
+Check with your service provider to see if they support SAML Single Logout or OIDC front-channel/back-channel logout. Not all providers support these features.
+:::
+
+## How Single Logout works in authentik
+
+When a user logs out or their session is terminated in authentik, the following process occurs:
+
+1. **Session Termination**: The user's session ends through one of the following:
+    - User-initiated logout via a logout flow.
+    - Administrative action (session deletion or user deactivation).
+    - Token revocation or expiration.
+2. **Provider Identification**: authentik identifies all OAuth2/OIDC and SAML providers with active sessions for the user that have SLO configured.
+3. **Logout Request Dispatch**:
+    - **Back-channel**: HTTP POST requests are sent directly from the authentik server to each back-channel provider's configured logout endpoint.
+    - **Front-channel**: For user-initiated logouts, a logout stage is automatically injected into the flow that handles browser-based logout (typically via iframes or sequential redirects).
+4. **Provider Processing**: Each provider processes the logout request, validates it, and terminates the user's active session.
+5. **Completion**: Once all providers have been notified, the user is redirected back to the authentik login screen.
+
+## Front-Channel vs. Back-Channel Logout
+
+authentik supports both front-channel (browser-based) and back-channel (server-to-server) logout methods, depending on how each provider is configured.
+
+### Front-Channel Logout
+
+Front-channel logout sends logout requests through the user's browser. authentik supports two front-channel modes:
```

Comment:
Ideally the two options, the two modes, would both be indented, since there are "children", listed items after the colon.